pkgdb2/templates/package.html

```diff
@@ -68,15 +68,15 @@
         <td colspan="2" align="center">
           <div id="monitoring_block">
             <input type="radio" id="no_monitoring" name="monitoring" value="0" />
-                <label for="no_monitoring" {%-
+                <label for="no_monitoring" title="Will toggle the monitor flag to monitor package for new releases" {%-
                     if package.monitoring_status == False %} class="active"{% endif
                     -%}>No monitoring</label> <br />
             <input type="radio" id="monitoring" name="monitoring" value="1" />
-                <label for="monitoring" {%-
+                <label for="monitoring" title="Will toggle the monitor flag to monitor bugs and the buildsystem for new builds" {%-
                     if package.monitoring_status == True %} class="active"{% endif
                     -%}>Bugs &amp; builds</label> <br />
             <input type="radio" id="no_build" name="monitoring" value="nobuild" />
-                <label for="no_build" {%-
+                <label for="no_build" title="Will toggle the monitor flag to monitor bugs for the package" {%-
```

Comment:
And finally: `Turns on monitoring of new releases but without attempt to build it`

What do you think?